stix/query.py

```diff
@@ -0,0 +1,230 @@
+#
+# Copyright (C) 2022 Vaticle
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import logging
+
+from stix.type_mapping import stix_entity_to_typedb, stix_attributes_to_typedb, stix_relation_to_typedb
+
+
+def sanitise_string(string_value):
+    return string_value.replace("'", "")
+
+
+class StixInsertGenerator:
```

Comment:
Generator for all the TypeDB insert queries

Reply:
this class has changed a relatively large amount!

Top things I noticed when refactoring:

1. clarified naming to use `stix` where it applies
2. removed any places where exceptions were being used to handle missing keys -- better to use an `if` check, then proceed if the key exists
3. can use `set()` like a `[]` in most places, no need to collec to to a `[]`, then turn it into a `set()`
4. the python string formatting with `f"...{program_variable}..."` makes generating queries a bit easier!

some other ones but those are the top line things we could take into future code